tsconfig.json

```
@@ -18,8 +18,10 @@
         "noUnusedLocals": true,
         "noUnusedParameters": true,
         "checkJs": true,
-        "outDir": "./tmp",
-        "noEmitOnError": true
+        "noEmit": true,
+        "noEmitOnError": true,
```

I believe `"noEmitOnError": true` is made redundant by the more broad `"noEmit": true`

```suggestion
        "noEmit": true,
```